src/main/java/data/repositories/UsuarioRepository.java

```diff
@@ -1,8 +1,8 @@
 package data.repositories;
 
 import data.entities.Usuario;
-import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.jpa.repository.JpaRepository;
```

Comment:
Why JPA and not CrudRepository ?

Reply:
Estoy utilizando JPA, que en sí encapsula a CrudRepository, para poder utilizar los métodos `.save()` , `.findById()` del objeto UsuarioRepository. Al hacer esto, evito crear los métodos `findAll()` , `findOne()` del  UsuarioService.java. Es por ello que dichos métodos de UsuarioService están comentados.

### Fuente:

- [SpringBoot API Doc](https://spring.io/guides/tutorials/bookmarks/)
- [StackOverflow - What is difference between CrudRepository and JpaRepository interfaces in Spring Data JPA?](https://stackoverflow.com/questions/14014086/what-is-difference-between-crudrepository-and-jparepository-interfaces-in-spring)